openreview/venue_request/process/matchingProcess.py

```diff
@@ -36,21 +36,22 @@ def process(client, note, invitation):
     print(traceback.format_exc())
 
     comment_note = openreview.Note(
-        invitation = note.invitation.replace('Paper_Matching_Setup', 'Comment'),
+        invitation = note.invitation.replace('Paper_Matching_Setup', 'Paper_Matching_Setup_Status'),
         forum = note.forum,
         replyto = note.id,
         readers = [conference.get_program_chairs_id()] + [SUPPORT_GROUP],
         writers = [],
         signatures = [SUPPORT_GROUP],
         content = {
-            'title': 'Matching Status [{note_id}]'.format(note_id=note.id),
+            'title': 'Paper Matching Setup Status',
             'comment': ''
         }
     )
 
     if matching_status.get('error'):
-        error_status = f'''{len(matching_status.get('error'))} error(s): {matching_status.get('error')}'''
-        comment_note.content['comment'] += error_status
+        error_status = f'''{len(matching_status.get('error'))} error(s): 
```

Comment:
if there could be more than one error,`errors_status` could be an array of strings.

Reply:
The paper matching status shouldn't have this text `To check references for the note: https://api.openreview.net/references?id={note.id}`